resources/keywords/navigation.resource

```diff
@@ -47,24 +47,40 @@ User Navigates To DBaaS Inventories Screen
     Wait Until Page Contains Element    ${elemDbaasStatus}    timeout=15s
     Page Should Contain Element    ${lnkProvAcc}
     Click Element    ${lnkProvAcc}
-    Wait Until Page Contains Element    ${titleDbaasInventories}    timeout=10s
 
 User Navigates To DBaaSPolicy Screen
-    [Arguments]    ${project}=${operatorNamespace}
+    [Arguments]    ${PROJECT}=${operatorNamespace}
     User Selects Administrator View
     User Navigates To Installed Operators Under Operators
-    User Filters ${project} Namespace On Project Dropdown
+    User Filters ${PROJECT} Namespace On Project Dropdown
     Wait Until Page Contains Element    ${elemDbaasStatus}    timeout=15s
     Wait Until Keyword Succeeds    5x    3s    Run Keywords
     ...    User Navigates To Database Access Under Data Services
     ...    AND    User Navigates To Installed Operators Under Operators
     ...    AND    Element Should Be Visible    ${lnkDBaaSPolicy}    timeout=10s
     Click Element    ${lnkDBaaSPolicy}
-    Wait Until Page Contains Element    ${titleDbaasPolicies}    timeout=10s
+    ${version}=     Run Keyword And Return Status    Wait Until Page Contains Element    ${titleDbaasPolicies}    timeout=10s
+    IF  not ${version}
+        Element Should Be Visible    ${titleDbaasPolicys}
+        Log    Cluster version is 4.11, Known issue
+    ELSE
+        Fail    msg=DBaaSPolicy page not loaded
+    END
 
 User Filters ${projectName} Namespace On Projects Screen
     Wait Until Page Contains Element    ${txtProjects}    timeout=10s
     Element Should Be Visible    ${txtBxSearchByName}
     Input Text    ${txtBxSearchByName}    ${projectName}
     ${lnkProjectNameUpd} =    Replace String    ${lnkProjectName}    <<project>>    ${projectName}
     Wait Until Page Contains Element    ${lnkProjectNameUpd}    timeout=5s
+
+User Is On DBaaSInventory Screen
+   ${version_410}=     Run Keyword And Return Status    Wait Until Page Contains Element    ${titleDbaasInventories}    timeout=10s
```

Comment:
Can't we use the same ${version} across all the version verifications?

Reply:
We could and it wont do any harm as the script overrides the value. But I felt it would be more readable if we have different variables for different versions as we would be aware this is something to be removed in future. So, Please let me know if this is fine or I could make changes.

Reply:
We are going to have a lot of new versions in the future. Maybe it is best to use one? LMKWYT

Reply:
I agree, but shall we come back to this once we have the newer versions?

Reply:
OK sure. Let's just not forget about it. Maybe create a task for it?